clodius/tiles/bigbed.py

```diff
@@ -0,0 +1,407 @@
+import bbi
+import functools as ft
+import logging
+import numpy as np
+import pandas as pd
+import re
+import sys
+import random
+
+from concurrent.futures import ThreadPoolExecutor
+
+MAX_THREADS = 4
+TILE_SIZE = 1024
+MIN_ELEMENTS = 1
+MAX_ELEMENTS = 50
+
+logger = logging.getLogger(__name__)
+
+range_modes = {}
+range_modes['significant'] = {'name': 'Significant', 'value': 'significant'}
```

Comment:
The PR description says `significance` and one of the tile queries below also uses `significance`. Is it intentional to use `significant` here?

Reply:
I think I have a typo in my PR description. I use `significant` in the module and in the associated test.